seed/challenges/01-front-end-development-certification/bootstrap.json

```diff
@@ -507,9 +507,9 @@
       "id": "bad87fee1348cd8acef08811",
       "title": "Taste the Bootstrap Button Color Rainbow",
       "description": [
-        "The <code>btn-primary</code> class is the main color you'll use in your app. It is useful for highlighting actions you want your user to take.",
-        "Add Bootstrap's <code>btn-primary</code> class to your button.",
-        "Note that this button will still need the <code>btn</code> and <code>btn-block</code> classes."
+        "The <code>btn-primary</code> class styles app elements you apply it to in order to indicate the primary action on a page. It essentially highlights actions you want your user to take.",
```

Comment:
I think at this point we've already established that a class styles the elements it is applied to. To me, the sentence without comma's makes "styles app elements you apply it on" a part of "in order to indicate the primary action". With comma's, "styles the app elements you apply it on" seems like a separate part explaining what a class does.
I agree that this sentence could probably be improved. I'll take a look at what I can do tomorrow.